<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SelectThunderbirdFilterFilesWidget</class>
 <widget class="QWidget" name="SelectThunderbirdFilterFilesWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QRadioButton" name="selectFile">
     <property name="text">
      <string>Select File:</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">buttonGroup</string>
     </attribute>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="KUrlRequester" name="fileUrl"/>
   </item>
   <item row="1" column="0">
    <widget class="QRadioButton" name="selectFromConfig">
     <property name="text">
      <string>Select file from config:</string>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">buttonGroup</string>
     </attribute>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="KComboBox" name="profiles"/>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QListWidget" name="listFiles"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KComboBox</class>
   <extends>QComboBox</extends>
   <header>kcombobox.h</header>
  </customwidget>
  <customwidget>
   <class>KUrlRequester</class>
   <extends>QFrame</extends>
   <header>kurlrequester.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
 </buttongroups>
</ui>
